entrypoint.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-set -eux -o pipefile
+set -ex -o pipefail
```

Comment:
Oopss!

I was also thinking to update the output to remove `-x` and have user understandable `echo` statements. WDYT? Currently its too hard to know what's happening for a user.